mlrun/serving/remote.py

```diff
@@ -68,24 +91,36 @@ def __init__(
                             this require that the event body will behave like a dict, example:
                             event: {"x": 5} , result_path="resp" means the returned response will be written
                             to event["y"] resulting in {"x": 5, "resp": <result>}
+        :param retries:     number of retries (in exponential backoff)
+        :param backoff_factor: A backoff factor in secounds to apply between attempts after the second try
+        :param timeout:     (optional) How long to wait for the server to send data before giving up, float in seconds
```

Comment:
Aren't these all optional, no just timeout?

Reply:
yes, i will remove the note